README.md

```diff
@@ -430,6 +430,7 @@ These variables must be set at the root or on a container element wrapping any b
 ```css
 html {
   --bc-color-brand: #196ce7; /* Only 6-digit hex and rgb formats are supported! */
+  --bc-color-brand-text: #ffffff; /* override text color for buttons coloured in brand colour. Default is otherwise chosen between white and black based on luminance of the colour defined in --bc-color-brand*/
```

Comment:
Could this be moved into the section below ("Optional CSS variables for further customization")

Reply:
Yes it might fit better there in optional? I was wondering a bit as in a sense they are all optional, as its not needed to customise --bc-color-brand either.

Reply:
Good point, I think we can merge these two sections then